test/new-e2e/pkg/runner/parameters/store_config_file.go

```diff
@@ -103,6 +107,10 @@ func (s configFileValueStore) get(key StoreKey) (string, error) {
 		value = s.config.ConfigParams.AWS.KeyPairName
 	case PublicKeyPath:
 		value = s.config.ConfigParams.AWS.PublicKeyPath
+	case PrivateKeyPath:
+		value = s.config.ConfigParams.AWS.PrivateKeyPath
+	case PrivateKeyPassword:
```

Comment:
<!-- Suggestions propose improvements to the current subject. It’s important to be explicit and clear on what is being suggested and why it is an improvement. -->
**💬 suggestion**
You could add a test to `store_config_file.test`, you can update the fixture file at  `testfixtures/test_config_with_stackparams.yaml` and add an assert to `Test_NewConfigFileStore`